lib/cdo/delete_accounts_helper.rb

```diff
@@ -124,11 +124,12 @@ def clean_and_destroy_pd_content(user_id)
     end
   end
 
-  # Anonymizes the user by deleting various pieces of PII and PPII from the User and UserGeo models.
+  # Anonymizes the user by deleting various pieces of PII and PPII
   # @param [User] user to be anonymized.
   def anonymize_user(user)
     UserGeo.where(user_id: user.id).each(&:clear_user_geo)
     SignIn.where(user_id: user.id).destroy_all
+    AuthoredHintViewRequest.where(user: user).each(&:clear_level_source_associations)
```

Comment:
Should this live in `clean_level_source_backed_progress`?

Reply:
Yes, that might be a better place for it.  I'll move it in a follow-up PR.

It's not clear to me what belongs in `anonymize_user` and what belongs in `purge_user` (which calls `anonymize_user`).  I'm not sure it matters much, really - as long as it's tested.

I don't actually understand how `authored_hint_view_requests.prev_level_source_id` et al are potentially identifying information, but [the spec](https://docs.google.com/spreadsheets/d/1RybuZunt5F6zSXp33oC4KdaBl20VMHVi0yprI8P30y0/edit#gid=670705846) I was given specifies that they should be cleared.